WikiClone/Article.html

```diff
@@ -0,0 +1,77 @@
+<! DOCTYPE html>
+<head xmlns="http://www.w3.org/1999/html">
+	<title>Legionary</title>
+	<link href="css/Main.css" type="text/css" rel="stylesheet">
+	<link href="css/Details.css" type="text/css" rel="stylesheet">
+</head>
+<body>
+<header>
+	<ul class="header_navbar">
+		<li class="navbar_item"><a href="Main.html"><p class="title">WikiClone</p></a></li>
+		<li class="navbar_item" style="float:right"><a href="Register.html" class="header_link">Register</a></li>
+		<li class="navbar_item" style="float:right"><a href="Login.html" class="header_link">Login</a></li>
+	</ul>
+</header>
+<main>
+	<div class="content">
+		<div class="container_name">
+			<div class="container">
+				<div class="item_left">
+					<h1>
+						Legionary
+					</h1>
+				</div>
+				<div class="item_right">
+					<a class="button_site">Remove</a>
+				</div>
+				<div class="item_right">
+					<a href="EditArticle.html" class="button_site">Edit</a>
```

Comment:
Same question here.

Reply:
Here should be a link because I have special page for editing.